xbmc/rendering/dx/RenderSystemDX.cpp

```diff
@@ -1915,6 +1914,13 @@ void CRenderSystemDX::FixRefreshRateIfNecessary(const D3D10DDIARG_CREATERESOURCE
   }
 }
 
+void CRenderSystemDX::GetRefreshRatio(uint32_t refresh, uint32_t * num, uint32_t * den)
```

Comment:
I think that it will not be apparent to the caller that the num and den will be changed in method. passing pointers is common practices for DX API. but if you insist I'll change.